### ISSUE TITLE
SapiEmitter should emit also duplicate headers

### ISSUE THREAD

**Issue by roxblnfk** [yiisoft/yii-web#178]
### What steps will reproduce the problem?
```php
$response
    ->withHeader('X-Test', '1')
    ->withAddedHeader('X-Test', '2')
    ->withAddedHeader('X-Test', '3; 3.5')
    ->withAddedHeader('X-Test', '4');
$emitter->emit($response);
```

### What is the expected result?
Headers:
```
X-Test: 1
X-Test: 2
X-Test: 3; 3.5
X-Test: 4
```

### What do you get instead?
Headers:
```
X-Test: 4
```

In addition, we should decide whether to use [header_remove()](https://github.com/yiisoft/yii-web/blob/13b097a1afa6c1b18ac53ab1ef327bee3b929c45/src/Emitter/SapiEmitter.php#L25) before sending ResponseInterface headers.

**Comment by samdark** [comment]
> In addition, we should decide whether to use header_remove() before sending ResponseInterface headers.

**Comment by roxblnfk** [comment]
> I think we should use it because we should emit what's in response object, nothing more.

I am talking about cases where the user in his code (or in the extension code) used `header()` instead of `$ response-> withHeader()`\
Emitter will emit only Response instance headers

**Comment by samdark** [comment]
Yes. Correct.

**Comment by samdark** [comment]
Fixed by #175
